### PR TITLE
added about page

### PR DESCRIPTION
**Pull request by sanggwonjang**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -18,24 +19,82 @@ class Interprep extends StatelessWidget {
...
+List<String> aboutInfo = AboutInfo.aboutInfo;
+
+List<Widget> widgets = aboutInfo
+    .map((str) => Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: (str[0] == '@')
+              ? new Text(str, style: TextStyle(fontWeight: FontWeight.bold))
+              : new Text(str),
+        ))
+    .toList();
```
Comment (hyeonwooshim) [inline lib/main.dart:69]: Can you put this inside `AboutInterface`, and `widgets` is too vague of a name

Comment (sanggwonjang) [reply lib/main.dart]: fixed

File: lib/main.dart [hunk 2]
```diff
@@ -18,24 +19,82 @@ class Interprep extends StatelessWidget {
...
+            onPressed: null,
+          ),
+        ],
+      ),
+      body: CardInterface(),
+    );
+  }
+}
+
+List<String> aboutInfo = AboutInfo.aboutInfo;
```
Comment (hyeonwooshim) [inline lib/main.dart:60]: this too

Comment (sanggwonjang) [reply lib/main.dart:60]: Should they go inside void initState() or something? Or just inside `AboutInterface`

Comment (hyeonwooshim) [reply lib/main.dart:60]: It's a constant so I don't think you even need to assign it to variable. Just use it directly.

**Pull request merged**